script.js

```diff
@@ -16,3 +16,83 @@ listItem.forEach((item) => {
     mainMenu.classList.remove('menushow');
   });
 });
+
+const placeholders = [
+  {
+    title: 'Multi-Post Stories',
+    subtitle: 'Gain+Glory',
+    tags: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+  },
+  {
+    title: 'Multi-Post Stories',
+    subtitle: 'Gain+Glory',
+    tags: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+  },
+  {
+    title: 'Multi-Post Stories',
+    subtitle: 'Gain+Glory',
+    tags: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+  },
+  {
+    title: 'Multi-Post Stories',
+    subtitle: 'Gain+Glory',
+    tags: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+  },
+  {
+    title: 'Multi-Post Stories',
+    subtitle: 'Gain+Glory',
+    tags: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+  },
+  {
+    title: 'Multi-Post Stories',
+    subtitle: 'Gain+Glory',
+    tags: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+  },
+];
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
Kindly remove all lines which disable linter checks in your project and try to solve them yourself. Learning to solve linter errors will help your learn best coding practices and make you a better developer👍